control-plane/pkg/apis/internals/kafka/eventing/v1alpha1/consumer_types.go

```diff
@@ -19,10 +19,12 @@ package v1alpha1
 import (
```

Comment:
just curious; do we want to have manifests for our internal APIs, at some point
(see https://github.com/knative/serving/blob/main/config/core/300-resources/podautoscaler.yaml#L20)

Reply:
Of course

Reply:
Done https://github.com/knative-sandbox/eventing-kafka-broker/pull/1637